lib/kennel.rb

```diff
@@ -84,11 +85,11 @@ def filter
     end
 
     def syncer
-      @syncer ||= Syncer.new(api, generated, project_filter: filter.project_filter, tracking_id_filter: filter.tracking_id_filter)
+      @syncer ||= Syncer.new(api, generated, kennel: self, project_filter: filter.project_filter, tracking_id_filter: filter.tracking_id_filter)
     end
 
     def api
-      @api ||= Api.new(ENV.fetch("DATADOG_APP_KEY"), ENV.fetch("DATADOG_API_KEY"))
```

Comment:
nit: I kinda liked "Env stuff / global setup happens in kennel.rb", but not sure if that is still true

Reply:
ENV is read all over the place though. If there was a pattern that ENV is only read from kennel.rb then it's passed me by.

Reply:
yeah I think over time this changed ... simpler to have it in api 👍